moped-editor/package.json

```diff
@@ -1,10 +1,14 @@
 {
   "name": "atd-moped-editor",
   "author": "ATD Data & Technology Services",
-  "license": "CC0 1.0 Universal",
+  "license": "CC0 1.0",
```

Comment:
`npm WARN atd-moped-editor@1.2.0 license should be a valid SPDX license expression`

https://spdx.org/licenses/ The expression drops Universal